model_cards/lit.md

```diff
@@ -0,0 +1,215 @@
+# Model Card: LiT (Locked image Tuning)
+
+Last updated: 2022-06-19
+
+Version: 1.0
+
+- This doc: https://github.com/google-research/vision_transformer/blob/main/model_cards/lit.md
+- Model Page: https://github.com/google-research/vision_transformer#lit-models
+- Other Links:
+  [LiT Blogpost](https://ai.googleblog.com/2022/04/locked-image-tuning-adding-language.html),
+  [LiT Paper],
+  [LiT Demo](https://google-research.github.io/vision_transformer/lit/)
+
+A text/image input model that can be used to embed text/image individually,
+and compute similarities between embeddings of text/image pairs. This enables
+use cases like zeroshot classification, or image/text retrieval.
+
+Note that this model card refers to the models that have been released on
+Github specifically (B16B, L16L). The [LiT Paper] also evaluates models that
```

Comment:
The LiT Colab enlists four models. Let's include their names too for clarity?

Reply:
> The [LiT Paper] also evaluates models that

Potentially missing link?

Reply:
Added a comment about the Colab models.

wrt `[LiT Paper]`: Markdown allows you to specify links separately for improved readability (see the `[LiT Paper]: https://arxiv.org/abs/2111.07991` later in the paragraph) - BTW you can preview the modelcard here:  https://github.com/google-research/vision_transformer/blob/test_455908256/model_cards/lit.md